src/Symfony/Component/Validator/Constraints/CountValidator.php

```diff
@@ -41,24 +41,28 @@ public function validate($value, Constraint $constraint)
         $count = \count($value);
 
         if (null !== $constraint->max && $count > $constraint->max) {
-            $this->context->buildViolation($constraint->min == $constraint->max ? $constraint->exactMessage : $constraint->maxMessage)
+            $exactlyOptionEnabled = $constraint->min == $constraint->max;
```

Comment:
Why not === ?

Reply:
I don’t know.  I thought it was because the value can be either a int or a string.  I just saved the result to a variable

Reply:
@OskarStark It was like this before and I wouldn't change this behavior in the scope of this PR.

Reply:
Fine